packages/rich-text/src/plugins/EmbeddedEntityBlock/LinkedEntityBlock.tsx

```diff
@@ -49,9 +50,13 @@ export function LinkedEntityBlock(props: LinkedEntityBlockProps) {
       className={styles.root}
       data-entity-type={entityType}
       data-entity-id={entityId}
-      draggable={true}
-      contentEditable={false}>
-      <div>
+      // COMPAT: This makes copy & paste work for Firefox
+      contentEditable={!HAS_BEFORE_INPUT_SUPPORT ? false : undefined}
+      draggable={!HAS_BEFORE_INPUT_SUPPORT ? true : undefined}>
```

Comment:
So does it mean that the block won't be draggable if `beforeInput` is supported?

Reply:
Dragging is supported whether `beforeInput` is supported or not. This "hack" is for copy & paste solution only. Interestingly the `draggable` attribute plays a role here as well. I've tested both drag & drop and copy & paste on the browsers I've mentioned above.